src/Operation/EstimatedDocumentCount.php

```diff
@@ -90,18 +115,53 @@ public function __construct($databaseName, $collectionName, array $options = [])
      */
     public function execute(Server $server)
     {
-        return $this->count->execute($server);
+        $command = $this->createCommand($server);
+        if ($command instanceof Aggregate) {
```

Comment:
Given that we expose the command document using `getCommandDocument`, I thought it was best to create the command and attach special handling to the `aggregate` command, as opposed to checking for the correct wire version here again. Let me know if you'd prefer an explicit wire version check here instead of the hidden check in `createCommand`.

Reply:
I realize `getCommandDocument` complicates things a bit. I think this is fine.

I'd probably just suggest a blank line between assigning `$command` and the `if` for readability.